Changelog

```diff
@@ -1,4 +1,81 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add `Auto resize columns` functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in `Manual` mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when default "temp" path changed (glassez)
+    - FEATURE: Add tuning options related to performance warnings (Chocobo1)
+    - FEATURE: Add right click menu for status filters (An0n)
+    - FEATURE: Allow setting the number of maximum active checking torrents (An0n)
+    - FEATURE: Add option to toggle filters sidebar (AbeniMatteo)
+    - FEATURE: Allow to set `working set limit` on non-Windows OS (Chocobo1)
+    - FEATURE: Add `Export .torrent` action (Chocobo1)
+    - FEATURE: Add keyboard navigation keys (itlezy)
+    - FEATURE: Allow to use POSIX-compliant disk IO type (Coda)
+    - FEATURE: Add `Filter files` field in new torrent dialog (thalieht)
+    - FEATURE: Implement new icon/color theme (now-im)
+    - FEATURE: Add file name filter/blacklist (mxtsdev, thalieht)
+    - FEATURE: Add support for custom SMTP ports (Emil M George)
+    - FEATURE: Split the OS cache settings into Disk IO read/write modes (summer)
+    - FEATURE: When duplicate torrent is added set metadata to existing one (glassez)
+    - FEATURE: Greatly improve startup time with many torrents (glassez)
+    - FEATURE: Add keyboard shortcut to Download URL dialog (Chocobo1)
+    - FEATURE: Add ability to run external program on torrent added (glassez)
+    - FEATURE: Add infohash and download path columns (tristanleboss)
+    - FEATURE: Allow to set torrent stop condition (glassez, thalieht)
+    - FEATURE: Add a `Moving` status filter (tristanleboss)
+    - FEATURE: Change color palettes for both dark, light themes (Chocobo1)
+    - FEATURE: Add a `Use proxy for hostname lookup` option (Nathan Lewis)
+    - FEATURE: Introduce a `change listen port` cmd option (BallsOfSpaghetti)
+    - BUGFIX: Store hybrid torrents using `torrent ID` as basename (glassez)
+    - BUGFIX: Enable Combobox editor for the `Mixed` file download priority (Aleksandr Cupacenko)
+    - BUGFIX: Allow shortcut folders for the Open and Save directory dialogs (Aleksandr Cupacenko)
+    - BUGFIX: Rename content tab `Size` column to `Total Size` (Aleksandr Cupacenko)
+    - BUGFIX: Fix scrolling to the lowermost visible torrent (Aleksandr Cupacenko)
+    - BUGFIX: Allow changing file priorities for finished torrents (An0n)
+    - BUGFIX: Focus save path when Manual mode is selected initially (Aleksandr Cupacenko)
+    - BUGFIX: Disable force reannounce when it is not possible (An0n)
+    - BUGFIX: Add horizontal scrolling for tracker list and torrent content (NotTsunami)
```

Comment:
I would call this a feature, as it was strictly adding new code, not fixing existing code

Reply:
Sure. But it is kinda of a "bug" to not have horizontal scrolling in the first place. I can hardly call it a "feature".